resources/assets/uad_lists.json

```
@@ -35748,5 +35748,581 @@
+    "list": "Oem",
+    "description": "'Security tools' name app only found",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.xiaomi.touchservice",
+    "list": "Oem",
+    "description": "No activities, uses miui analytics looks like a tracking touch.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "cn.wps.moffice_eng.xiaomi.lite",
+    "list": "Oem",
+    "description": "Chinese WPS Office",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "eu.xiaomi.ext",
+    "list": "Oem",
+    "description": "Xiaomi.eu Extension\nCalendar app things. Only for China.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "eu.xiaomi.module.inject",
+    "list": "Oem",
+    "description": "Inject fields\nFields name of device info like which os version or security date.\nNot sure if it's needed.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  {
+    "id": "com.xiaomi.bttester",
+    "list": "Oem",
+    "description": "BTCIT\nBluetooth Test Service",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.iflytek.inputmethod.miui",
+    "list": "Oem",
+    "description": "Chinese Keyboard to Miui China, replace Keyboard before remove.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.xiaomi.ugd",
+    "list": "Oem",
+    "description": "GPU Driver Updater\nIt's weird when this app cameback on HyperOS(from MIUI 12).\nUpdates GPU driver.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Expert"
+  },
+  {
+    "id": "org.dtx.aidl.manager",
+    "list": "Oem",
+    "description": "DtxService\nAnt Dtx Service, unknown app, but safe to remove, probably useful in China.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.google.android.overlay.healthconnect",
+    "list": "Oem",
+    "description": "Useless overlay to 'com.google.android.apps.healthdata'.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.magiear.handsfree.assistant",
+    "list": "Misc",
+    "description": "Hands-Free Assistant\nOnly has permission activity voice from mediatek.\nhttps://play.google.com/store/apps/details?id=com.magiear.handsfree.assistant",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.miui.mediaviewer",
+    "list": "Oem",
+    "description": "Media viewer\nOld? Mi Video.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
```

```suggestion
    "removal": "Recommended"
```
Mi Video is recommended right? Then why this is not?

---

its because of its completly different version, no trackers app